src/index.ts

```diff
@@ -8,3 +10,15 @@ if ('SENTRY_DSN' in process.env
     environment: process.env.SENTRY_ENVIRONMENT,
   });
 }
+
+const listenOptions: ListenOptions = {
+  port: Number.parseInt(process.env.SSH_PORT ?? '22'),
+  host: process.env.SSH_HOST ?? '127.0.0.1',
+};
+
+server.listen(
+  listenOptions,
+  function () {
+    console.log(`Listening for SSH requests on ${listenOptions.host ?? ''}:${listenOptions.port ?? ''}`);
```

Comment:
We should probably use a real logger.

I'll set that up in another MR / issue.

Reply:
Opened in #11 -- if that gets merged before this I'll switch to it (that'd probably be the ideal order)